apps/airview-demo/src/views/document-view/external-content.js

```diff
@@ -20,5 +22,22 @@ export function ExternalContent({ metadata }) {
     getData();
   }, [setData, metadata]);
 
-  return <MDEditor.Markdown source={data} />;
+  const components = {
+    img: ({ src, alt }) => {
+      if (!src.startsWith("https:")) {
```

Comment:
We should account for http. There's a regex here already which does the same thing -https://github.com/AirWalk-Digital/airview/blob/main/packages/airview-cms/src/features/cms/body-editor/markdown-editor.js#L23

Reply:
Thanks for pointing out it. Have made the changes